src/chart/MultiChart.js

```diff
@@ -115,6 +115,19 @@
         this._chartTypeProperties = _;
         return this;
     };
+    
+    MultiChart.prototype.getChartDataType = function () {
+        var context = this;
+        var types = ["1D","2D","ND"];
+        for(var i=0;i<types.length;i++) {
+            var idx = "_"+types[i]+"ChartTypes";
+            for(var j=0;j<context[idx].length;j++) {
+                if(context[idx][j].id === context.chartType()){
+                    return types[i];
+                }
+            }
+        }
+    };
```

Comment:
Is it really a candidate for a performance bottleneck?

It does make more sense to move the logic to the location that the chart types are specified (...to prevent problems with future additions).